servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpServerBuilder.java

```diff
@@ -100,6 +101,16 @@ default HttpServerBuilder sslConfig(ServerSslConfig defaultConfig, Map<String, S
                 "sslConfig(ServerSslConfig, Map, int, Durations) is not supported by " + getClass());
     }
 
+    /**
+     * Allows to configure the listen mode when SSL is enabled.
+     *
+     * @param mode the mode to use.
+     * @return {@code this}.
+     */
+    default HttpServerBuilder sslListenMode(final SslListenMode mode) {
```

Comment:
I'm going to go with the first approach, because with the second one it's again ambiguous (the default config does not have the acceptInsecureConnections).